src/core/dom/layout/observers.d.ts

```diff
@@ -0,0 +1,11 @@
+export {};
+
+declare global {
```

Comment:
I'm opposed to that approach because that's how we ended up with a massive amp.externs.js file full of every type polyfill anyone ever needed, versus keeping polyfills adjacent to the file(s) that depend on them. There's not a lot of them, ex. this is the only `.d.ts` file in `#core/dom/layout` and there's less than ten other files in that directory. `#core/types/string/crypto.d.ts` likewise contains the `msCrypto` definition on the `Window` object, `#core/data-structures/weakref.d.ts` has one for `WeakRef` etc because that's where they're used. My gut feeling is this is more maintainable (and easier to trace usage/split up core later if desired) than a single `#core/polyfills.d.ts`.

CAVEAT: `src/polyfills` passed CC type-checking but hasn't been migrated to TS yet. Once it is, it'll make sense for that directory to house many of these polyfill declarations instead.